### ISSUE TITLE
Use model/genetic map etc IDs in the text

### ISSUE THREAD

**Issue by jeromekelleher** [popgensims/manuscript#42]
We're referring to models and genetic maps in the text as "Gutenkunst model" and "Comeron map" etc along with citations each time. This seems clunky to me for a few reasons:

1. We're assuming the reader knows how, .e.g, the Gutenkunst model differs from the Ragsdale model, which is unreasonable and hard to follow for non-experts
2. There's a *lot* of citing of the same 4 or 5 papers.

I think we can make things flow a lot better and get rid of a lot of citation cruft if we refer to these using the same IDs as we use in stdpopsim. They should be in a table that we have provided along with citations so that people can look them up. Furthermore, this is what we want people to do in the future when they are writing papers using stdpopsim, right? So, we should set the example.

**Comment by andrewkern** [comment]
so are you thinking that we would introduce the model fully once, and then give the acronym in the text, or just go straight to names like `OutOfAfrica_3G09` and point them to a table?

**Comment by jeromekelleher** [comment]
> so are you thinking that we would introduce the model fully once, and then give the acronym in the text, or just go straight to names like OutOfAfrica_3G09 and point them to a table?

Not sure --- getting the references into the table would be challenging in terms of horizontal space, so perhaps we introduce the models in the text once, and then refer to them by IDs afterwards?

**Comment by andrewkern** [comment]
yeah that would be my vote. I can give this a run this morning-- I'm planning on writing this manuscript most of the day.

**Comment by andrewkern** [comment]
@jeromekelleher you and I should settle on Table 1 at this point. Should we put the new IDs in?

**Comment by jeromekelleher** [comment]
Yep, I'll run the simulations and fix up the table/caption tomorrow. Will all of the models have been defined in the text? If so, I can drop the citations and get some valuable horizontal space, right?

**Comment by andrewkern** [comment]
What if we have two tables? one that has ID, description and reference, and a second that has the run time stats?

**Comment by jeromekelleher** [comment]
Lotta tables... leave it with me and I'll try out a few things. I'll be working full time on the doc tomorrow.

**Comment by andrewkern** [comment]
time to do the pass through the manuscript putting in demog_model IDs?

**Comment by jeromekelleher** [comment]
Definitely for the methods. Not clear whether we should do this in the results/discussion, as we'd need to the text quite a bit. See how it looks after doing the methods?

**Comment by andrewkern** [comment]
okay I can do that pass now

**Comment by jeromekelleher** [comment]
Although, looks like we need to do a bunch of rewriting in the results anyway (#95)

**Comment by andrewkern** [comment]
okay i can take a pass on this too

**Comment by andrewkern** [comment]
I believe this is now closed by https://github.com/popsim-consortium/manuscript/pull/118
